include/nighthawk/common/worker.h

```diff
@@ -30,6 +30,12 @@ class Worker {
    * and mandatory.
    */
   virtual void shutdown() PURE;
+
+  /*
+  Called on-thread after its designated task finishes. Last chance to clean up while the associated
```

Comment:
Nit: `* ` formatting at the start of each line.

Reply:
Done